ide/projectui/src/org/netbeans/modules/project/ui/ProjectChooserAccessory.java

```diff
@@ -174,6 +188,9 @@ public void actionPerformed( ActionEvent e ) {
         if ( e.getSource() == jCheckBoxSubprojects ) {
             OpenProjectListSettings.getInstance().setOpenSubprojects( jCheckBoxSubprojects.isSelected() );
         }
+        if (e.getSource() == jCheckBoxPrime) {
+            OpenProjectListSettings.getInstance().setTrustAndPrime(jCheckBoxPrime.isSelected());
```

Comment:
Is it OK that the settings here are remembered even if the user cancels the whole Project|Open at the end ? I know the 'Open Required Projects'  is working that way for ages, but remembering an action after Cancel is ... unexpected.

Reply:
Poor design by @hrebejk! The values are read in another class, but stored here.